proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyServer.java

```diff
@@ -142,6 +150,16 @@ public void start() {
 
     try {
       this.server.start();
+
+      // Schedule a task to log metrics at a fixed rate
+      StatisticsHandler stats = this.server.getChildHandlerByClass(StatisticsHandler.class);
+      this.scheduler.scheduleAtFixedRate(() -> {
+        log.debug("(jetty) Num requests: " + stats.getRequests());
+        log.debug("(jetty) Num requests: " + stats.getRequestsActive());
```

Comment:
Num active requests?

Reply:
ah thanks-- fixed it on the `sev-16337-add-header-logging` branch